releases/svx-1.2.4.md

```diff
@@ -0,0 +1,52 @@
+# SVX 1.2.4 Release Notes
+
+**Software Release Date**: Nov 7, 2023
+
+**Summary**:
+
+The following is a summary of the key features and enhancements:
+
+* **Enhancements**: Updated context switcher options for administrators depending on their administrator levels.
```

Comment:
@anastasiaUsik I don't understand what the updated context switcher does. Can you explain

Reply:
@javereec 
re-phrased proposal:
Improved the context-switching menu options based on the administrator levels

Explanation: If I'm only OA - I don't see a context switching menu, but when I've been added as a TA, this menu should appear for me with the option "return to Tenant dashboard", and etc

Reply:
Ok, now I understand